src/components/ElectionGuardBallotTrackingCode.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Election } from '@votingworks/ballot-encoder'
+import QRCode from './QRCode'
+
+import Prose from './Prose'
+import { NoWrap } from './Text'
+
+const TrackerPage = styled.div`
+  page-break-after: always;
+  @media screen {
+    display: none;
+  }
+`
+const PreHeader = styled.div`
+  margin: 0 auto 0.375in;
+  text-align: center;
+  white-space: nowrap;
+  font-size: 1.75rem;
+  font-weight: 600;
+`
+
+const TrackerBox = styled.div`
+  border: 0.2rem solid #000000;
+  padding: 2rem;
+`
+const TrackerID = styled.span`
+  line-height: 1;
+  font-family: monospace;
+  font-size: 2rem;
+`
+const Content = styled.div`
+  display: flex;
+  padding: 2rem 0;
+`
+const Footer = styled.div`
+  border-top: 0.1rem solid #000000;
+  padding-top: 1rem;
+`
+
+const Step = styled.p`
+  display: block;
+`
+const StepDetail = styled.span`
+  display: flex;
+  margin: 1rem 0 0 1rem;
+`
+const BrowserInstructions = styled.span`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 4rem;
+  border: 1px solid #000000;
+  padding: 1rem 1.5rem;
+  text-align: center;
+  line-height: 1.5;
+`
+
+const ElectionGuardBallotTrackingCode = ({
+  election,
+  tracker,
+}: {
+  election: Election
+  tracker: string
+}) => {
+  const { title, county, state, date } = election
+
+  const trackingDisplayUrl = 'https://fulton.electionguard.vote'
```

Comment:
Do you want to hard-code this URL?

Reply:
@ben, thoughts on moving this to election config JSON?

Reply:
eventually yes, but for WI, I want to limit the number of repos we change.